db/migrate/20190711155709_add_columns_to_pets.rb

```diff
@@ -0,0 +1,11 @@
+class AddColumnsToPets < ActiveRecord::Migration[5.2]
+  def change
+    add_column :pets, :record_numbers, :string
+    add_column :pets, :children_friendly, :boolean
+    add_column :pets, :allergic_friendly, :boolean
+    add_column :pets, :microchip, :boolean
```

Comment:
I would change the name of this column as well:
```suggestion
    add_column :pets, :microchipped, :boolean
```